src/Presentation/Http/Controllers/Web/README.md

```
@@ -0,0 +1,14 @@
+# Web Controllers
+
+Traditional web interface controllers:
+- HTML response generation
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines
null

(MD032, blanks-around-lists)